.github/workflows/cmake.yml

```diff
@@ -325,10 +326,15 @@ jobs:
         LSAN_OPTIONS: ${{ matrix.lsan-options || 'verbosity=1' }}
 
     - name: Upload coverage report
-      if: matrix.codecov
+      if: matrix.codecov && ( env.CODECOV_TOKEN_SECRET != '' || github.repository == 'zlib-ng/zlib-ng' )
```

Comment:
I think this should be `secrets.CODECOV_TOKEN != ''`

Reply:
It doesn't have access to secrets.CODECOV_TOKEN ...